common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -992,6 +992,13 @@ public final class LC {
     public static final KnownKey httpclient_mdm_devices_limit_per_server = KnownKey.newKey(50);
     public static final KnownKey httpclient_mdm_devices_total_limit= KnownKey.newKey(500);
 
+    public static final KnownKey lds_client_socket_timeout = KnownKey.newKey(45 * Constants.MILLIS_PER_SECOND);
+    public static final KnownKey lds_client_connection_request_timeout = KnownKey.newKey(30 * Constants.MILLIS_PER_SECOND);
+    public static final KnownKey lds_client_connection_timeout = KnownKey.newKey(25 * Constants.MILLIS_PER_SECOND);
+    public static final KnownKey lds_client_max_retries = KnownKey.newKey(3);
+    public static final KnownKey lds_client_max_connection = KnownKey.newKey(100);
+    public static final KnownKey lds_client_max_per_route = KnownKey.newKey(20);
+    public static final KnownKey lds_client_max_idle_time = KnownKey.newKey(20);
```

Comment:
```suggestion
    public static final KnownKey lds_client_socket_timeout_millis = KnownKey.newKey(45 * Constants.MILLIS_PER_SECOND);
    public static final KnownKey lds_client_connection_request_timeout_millis = KnownKey.newKey(30 * Constants.MILLIS_PER_SECOND);
    public static final KnownKey lds_client_connection_timeout_millis = KnownKey.newKey(25 * Constants.MILLIS_PER_SECOND);
    public static final KnownKey lds_client_max_retries = KnownKey.newKey(3);
    public static final KnownKey lds_client_max_connection = KnownKey.newKey(100);
    public static final KnownKey lds_client_max_per_route = KnownKey.newKey(20);
    public static final KnownKey lds_client_max_idle_seconds = KnownKey.newKey(20);
```
I think it would be better for future clarity if we included the time units in the LC names.

Reply:
Fixed